client/src/featureform/register.py

```diff
@@ -1582,31 +1580,29 @@ def get_source(self, name, variant, local=False):
                 path="",
             )
         else:
-            fakeDefinition = PrimaryData(location=SQLTable(name=""))
-            fakeSource = Source(
+            mock_definition = PrimaryData(location=SQLTable(name=""))
```

Comment:
maybe add a comment why we need this in the first place, to see non test code have a mock def(n) is pretty confusing.

emptyDefinition seems more suitable to me

Reply:
prep for future work. we're going to fill these guys in in the coming weeks.